src/shared/components/Wrapper/Styles.js

```diff
@@ -2,8 +2,8 @@ import styled from 'styled-components';
 import { colour } from '../../styles/StyleConstants';
 
 export const StyledContainer = styled.div`
-  width: 100vw;
-  max-width: 1200px;
+  width: 99vw;
+  ${'' /* max-width: 1200px; */}
```

Comment:
what is happening here?

Reply:
So that the site doesn't scroll right and left